sdcm/tester.py

```diff
@@ -478,7 +478,8 @@ def setUp(self):
                 self.legacy_init_nodes(db_cluster=self.db_cluster)
             else:
                 self.init_nodes(db_cluster=self.db_cluster)
-            if self.params.get('use_ldap_authorization') and not self.params.get('are_ldap_users_on_scylla'):
+            if (self.params.get('use_ldap_authorization') or self.params.get('use_ms_ad_ldap')) and not \
+                    self.params.get('are_ldap_users_on_scylla'):
```

Comment:
Can I use self.localhost.add_ldap_entry() to add user in MS AD?

-> LocalHost( *, LdapContainerMixin)

If the LdapContainerMixin is general, can we rename it to `LdapCommonMixin`?

Reply:
@amoskong , could you please expain that to me what you mean with the `LocalHost(*, LdapContainerMixin)?
if i understand it correctly your question, we don't need LocalHost calls for this, as we are not creating any tunnel to the MS AD server, but the real address and port (different from tests that run within our monitor node, and nodes from outside the same VPC will not be able to access it without the reverse tunnel.

the thing with our MS AD is that it is not dynamically created, so if you do changes to it, they will be persisted for other tests as well, so you must either ensure you clean after yourself (to not explode the instance's capacity) or the not fail anything, but yes, theoretically speaking (didn't test) `self.localhost.add_ldap_entry()` should allow you to add entries to it.

Reply:
The add_ldap_entry() is used to add ldap object/entry to ldap server.
Can I use it to add users in MS AD?  Or it can only be used to add entry to ldap container?

Reply:
as i said in my comment above, i have never tested this scenario, and it might work, but we need to pay attention to it, as opposite from the openldap containers, our MS AD is not dynamically created, and we will need to be sure we maintain it "clean" after we make changes during our tests